index.html

```diff
@@ -1,19 +1,121 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
-
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header>
+      <nav>
+        <img
+          src="./img/karma-logo.svg"
+          alt="karma-logo"
+          class="logo"
+          width="30px"
+        />
+        <ul class="nav-links">
+          <li>
+            <a href="#meet_karma"><strong>Meet Karma</strong></a>
+          </li>
+          <li><a href="#how_it_works">How it Works</a></li>
+          <li><a href="#store">Store</a></li>
+          <li><a href="#blog">Blog</a></li>
+          <li><a href="#help">Help</a></li>
+          <li><a href="#login">Login</a></li>
+        </ul>
+      </nav>
+      <div class="container">
+        <div class="image-content">
+          <p>Introducing Karma</p>
+          <p>Bring WiFi with you, everywhere you go.</p>
+          <button class="btn">Learn More</button>
+        </div>
+      </div>
+    </header>
+
+    <section>
+      <p class="title">Everyone needs a little Karma.</p>
+      <div class="how-container">
+        <div class="how_it_works">
+          <img
+            src="./img/icon-devices.svg"
```

Comment:
Instead of using padding to separate some centered items, on the "how container" class you can change` justify-content: center` to `space-around`. This will let you delete the left and right paddings.